src/main/java/com/splunk/logtosplunk/loggable_events/LoggableBlockEvent.java

```diff
@@ -0,0 +1,146 @@
+package com.splunk.logtosplunk.loggable_events;
+
+import com.splunk.logtosplunk.actions.BlockEventAction;
```

Comment:
It really feels to me like the "EventActions" are data of the "Event" I realize kinda commented on this already but the enum and data derived really kinda fits in here well.